### PR TITLE
fix: Update e2e tests to check payment made on application submission…

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
… and C2 upload

### CODE REVIEW

File: e2e/fixtures/gatekeeping.json [hunk 1]
```diff
@@ -118,7 +118,7 @@
             "partyId": "3a8785c6-178d-4138-bd1a-5eea0debed25",
             "jobTitle": "Legal adviser",
             "partyType": "ORGANISATION",
-            "pbaNumber": "PBA1234567",
+            "pbaNumber": "PBA0082848",
```
Comment (KiranVarma02) [inline e2e/fixtures/gatekeeping.json:121]: why this change in multiple files?

Comment (akomisarek) [reply e2e/fixtures/gatekeeping.json:121]: This is the test pba which works for us on aat and provide "real" results.

File: e2e/pages/paymentHistory.page.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const {I} = inject();
+const config = require('../config');
+
+const paymentsLocator = '//span[text()="Processed payments"]';
+
+const buildPaymentLocator = amount => config.mockedPayment ? `${paymentsLocator}/../table//tr[.//td[text()="Success"]]` : `${paymentsLocator}/../table//tr[.//td[text()="${amount}"] and .//td[text()="Success"]]`;
```
Comment (tomaszpowroznik) [inline e2e/pages/paymentHistory.page.js:6]: config.mockedPayment is true on local env and false in pipeline. On local environment we use static wiremock mappings to simulate payment sercvice, thus fee is always same regardless of what has been actually requested to be paid. For that reason, when working with mocks, fee amount is not checked

File: e2e/fixtures/gatekeeping.json (hunk 1 above)
Comment (tomaszpowroznik) [reply e2e/fixtures/gatekeeping.json:121]: potentially can be moved to config, but it requires refactoring data json files like this, as we cannot inject config value directly into json

**Review by akomisarek — APPROVED** [review]
